### PR TITLE
New backups: Properties table

### PR DESCRIPTION
**Pull request by makingthematrix**

### CODE REVIEW

File: storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        propertiesDao = userDatabase.propertiesDao()
+    }
+
+    @After
+    fun tearDown() {
+        userDatabase.close()
+    }
+
+    @Test
+    fun givenAListOfEntries_whenAllKeyValuesIsCalled_thenAssertDataIsTheSameAsInserted(): Unit = runBlocking {
```
Comment (mejdoo) [inline storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt:38]: you are testing properties Dao not key value :)

File: storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        val numberOfItems = 3
+        val data = KeyValueTestDataProvider.listOfData(numberOfItems)
```
Comment (mejdoo) [inline storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt:40]: KeyValueTestDataProvider?

Comment (makingthematrix) [reply storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt:40]: Right. It worked becaue KeyValues and Properties are techicaly the same thing. Fixed.

File: storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt (hunk 1 above)
Comment (makingthematrix) [reply storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/property/PropertiesDaoTest.kt]: fixed

**Pull request merged**